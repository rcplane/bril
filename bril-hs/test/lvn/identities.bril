@main {
  one: int = const 1;
  two: int = const 2;

  sum: int = add one two;
  sum1: int = add two one;

  prod: int = mul one two;
  prod1: int = mul two one;

  print sum sum1 prod prod;
}
