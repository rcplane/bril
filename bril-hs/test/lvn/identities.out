@main {
  sum: int = const 3;
  sum1: int = const 3;
  prod: int = const 2;
  print sum sum1 prod prod;
}
